.changeset/light-llamas-remember.md

```diff
@@ -0,0 +1,6 @@
+---
+"providence-analytics": patch
+"@lion/ui": patch
+---
+
+Reformatted code with version 3 of prettier
```

Comment:
Should this even be a version bump? We just formatted the code but didn't change anything. What do you think?

Reply:
No, that is not needed i guess